tests/unit_tests/test_cli.py

```diff
@@ -80,7 +82,9 @@ class MyDevice:
 
 
 @responses.activate
-def test_get_plans(runner: CliRunner):
+def test_get_plans(
+    runner: CliRunner,
+):
```

Comment:
```suggestion
def test_get_plans(runner: CliRunner):
```

Does the IDE set this back again if you revert this?